src/minigames/rhythmMinigame.ts

```diff
@@ -0,0 +1,62 @@
+import { TextStyle, Text } from "pixi.js";
+import { Minigame } from "./minigame";
+
+export class RhythmMinigame extends Minigame {
```

Comment:
this is actually a pretty clean extension of the keyboardMinigame. Just extend it then override onKeyDown.